templates/layout/base.html

```diff
@@ -2,9 +2,17 @@
 <html class="no-js" lang="{{ locale_name }}">
     <head>
         <title>{{ head.title }}</title>
+
+        <link rel="dns-prefetch preconnect" href="//cdn8.bigcommerce.com" crossorigin>
```

Comment:
should these be hardcoded, or can/should we populate them via a variable somehow?

CC @chrisboulton

Reply:
They should be fed in via a variable somehow. We have various endpoints, which depend on store assignment or store environment.

Reply:
I am pretty sure we are seeding it for stapler to generate asset urls.

Reply:
I'll change this so the cdn paths come from the site settings, which is available in the theme context.